### DISCUSSION TITLE
oss-fuzz maintainers

### DISCUSSION THREAD

**Discussion by cujomalainey** [thesofproject/sof#7393] [Team Posts]
Hello SOF TSC,

I am looking to see if anyone else is interested in joining the OSS fuzz bug list. Currently it is only Ranjani and myself and the recent IPC refactor has generated a handful of bugs that I'm not sure we have time to fix as some of them appear to be fairly wonky. These bugs have disclosure deadlines so it is better fix them sooner rather than later. I am hoping to have 1-2 people from each contributing SoC group + google to help get better coverage as I open up more code to the fuzzer.

**Comment by lgirdwood** [comment]
@RanderWang @libinyang @mengdonglin fyi for SDL.
@cujomalainey I can join the list, I'm sure the IPC abstraction and IPC4 support will continue and add bugs as we go.

**Comment by cujomalainey** [comment]
Excellent, please provide google accounts i can add to the system.

**Comment by RanderWang** [comment]
wonderful! that is what we need

**Comment by cujomalainey** [comment]
ping on this, system uses monorail which authenticates through Google.

**Comment by lgirdwood** [comment]
@cujomalainey email sent
